packages/squiggle-lang/__tests__/helpers/helpers.ts

```diff
@@ -39,3 +39,15 @@ export function expectErrorToBeBounded(
   const error = distance / normalizingDenom;
   expect(error).toBeLessThanOrEqual(epsilon);
 }
+
+export function assertTag<T extends SqValue["tag"]>(
```

Comment:
This is cute. I might change name to ``assertHsTag``, but it's minor.

Reply:
Yeah, it's annoying that native Jest tests don't act as TypeScript guards.

There's some slow progress on Jest and Typescript about that: https://github.com/jestjs/jest/issues/9146, https://github.com/DefinitelyTyped/DefinitelyTyped/issues/41179, https://github.com/microsoft/TypeScript/issues/40562, but it's not quite there yet.

> I thought this referred to our Tag system at first.

We probably should rename `SqValue.tag` -> `SqValue.type`...

Reply:
Good point